tests/e2e/framework/kubernetes.go

```diff
@@ -429,6 +429,11 @@ func (k *KubeInfo) Teardown() error {
 		if err != nil {
 			return nil
 		}
+
+		if err := util.DeleteNamespace(k.Namespace, k.KubeConfig); err != nil {
+			log.Errorf("Failed to delete namespace %s", k.Namespace)
+			return err
```

Comment:
Might not want to return err here.  If the helm chart being deleted has the namespace manifest then the `HelmDelete` step would delete the ns and this `DeleteNamespace` would fail since the ns is already gone.  In that case propagating the err from the Teardown method would be an issue.